opensrp-fp/src/main/java/org/smartregister/fp/common/task/FPNextProgressDialogTask.java

```diff
@@ -0,0 +1,11 @@
+package org.smartregister.fp.common.task;
```

Comment:
This class does not seem to provide any more functionality to the superclass. It there a need to extend it?

Reply:
yeah right now it's nothing to do so removing that